_posts/2019-05-30-Review-of-RSE-community-at-C3DIS-2019.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Review of the RSE community at C3DIS"
+date:   2019-05-30 21:50:04 +1000
+categories: jekyll update
+---
+
+The Collaborative Conference on Computational and Data Intensive Science (C3DIS) 2019 was on from the 6th to the 10th of May. We asked for members of the RSE community to volunteer to help us run a Hacky Hour in the evening of Wednesday the 8th May and to also extend an invitation to the Friday the 10th all day RSE workshop.
+
+The Hacky Hour on Wednesday was good and we had a handful of RSEs providing support with about 12 people in total at the Hacky Hour. It was useful to engage with the other RSEs as well as being able to talk to people about their problems. The C3DIS organisers thought it was worthwhile to do again next year and the RSEs that attended agreed that we should do it next year.
+
+The RSE workshop on the Friday started off with us all introducing ourselves, explaining why we were here and explaining what we wanted to get out of the day. This ended up taking the whole first session, but it was really valuable to find out the different motivations for being there.
+
+After morning tea, we did an unconference based on the 2019 deliverables. We broke into 3 main groups to work on:
+- Profile writing
+- Face to face meetings
+- Manifesto / Business plan
+
+The result of the profile writing session was that 3 new profiles were created that were allowed to be disseminated widely.
+
+The result of the face to face meetings was that David Benn started to put together a plan for a RSE meeting in Adelaide.
```

Comment:
I forgot to mention this at the Steering Committee meeting but Daniel Collins (David's boss) showed interest in organising a Canberra meetup

Reply:
I will change 12 to 15 for the Hacky Hour